src/routes/chat/ChatHeader/ChatHeader.jsx

```diff
@@ -87,19 +93,50 @@ const ChatCreateOffer = ({ setOffer }) => {
 
 const ChatHeader = () => {
   const [offer, setOffer] = useState(true);
-
+  const {
+    sellerPrice,
+    buyerPrice,
+    sellerNumberOfShares,
+    buyerNumberOfShares
+  } = useSelector(state => state.chat.chatConversation);
+  const userType = useSelector(state => state.misc.userType);
+  const otherPartyUserType = getOtherPartyUserType({ userType });
+  const userPrice = getUserPrice({
```

Comment:
Same for this one and all the other functions, just 3 parameters will do, without transforming it to an object